.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Expected Behavior (short explanation)
+
+Please describe the behavior you are expecting
+
+# Current Behavior
+
+What is the current behavior?
+
+# Failure Information
+
+Please help provide information about the failure.
+
+## Steps to Reproduce
+
+1. step 1
+2. step 2
+3. you get it...
+
+## Context
+
+Please provide any relevant information about your setup. 
+
+* Inmanta version:
+* Operating System:
```

Comment:
Maybe add _python version_ to the list as well? What about PostgreSQL version?

Reply:
Added it in https://github.com/inmanta/inmanta/pull/2416/commits/4f10b989b54c0c76c2a13c81416026cef61a37ed

Reply:
What about PostgreSQL version @bartv ? Would that be relevant?

Reply:
yes

Reply:
Did it in https://github.com/inmanta/inmanta/pull/2416/commits/109c0803e5788a67f3414672083d954ea8283964